staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -433,6 +433,11 @@ func (e *Store) Create(ctx context.Context, obj runtime.Object, createValidation
 	if e.Decorator != nil {
 		e.Decorator(out)
 	}
+	if dryrun.IsDryRun(options.DryRun) {
+		if err := dryrun.RevertDryRunMetadata(obj, out); err != nil {
```

Comment:
(nit) remove repetition, and 

```suggestion
		if err := dryrun.ResetMetadata(obj, out); err != nil {
```